src/universal/modules/meeting/containers/MeetingUpdates/MeetingUpdatesContainer.js

```diff
@@ -78,25 +78,43 @@ const mapStateToProps = (state, props) => {
   const projects = makeProjectsByStatus(memberProjects);
   return {
     projects,
-    queryKey: teamMemberId
+    queryKey: teamMemberId,
+    canShowEmptyState: false
   };
 };
 
-const MeetingUpdatesContainer = (props) => {
-  if (!props.projects) {
-    return <LoadingView />;
+@connect(mapStateToProps)
+export default class MeetingUpdatesContainer extends Component {
+  static propTypes = {
+    gotoItem: PropTypes.func.isRequired,
+    gotoNext: PropTypes.func.isRequired,
+    canShowEmptyState: PropTypes.bool,
+    localPhaseItem: PropTypes.number.isRequired,
+    members: PropTypes.array.isRequired,
+    projects: PropTypes.object.isRequired,
+    queryKey: PropTypes.string,
+    team: PropTypes.object.isRequired
+  };
+
+  constructor(props) {
+    super(props);
+    this.state = {
+      canShowEmptyState: false
+    };
   }
-  return <MeetingUpdates {...props} />;
-};
 
-MeetingUpdatesContainer.propTypes = {
-  gotoItem: PropTypes.func.isRequired,
-  gotoNext: PropTypes.func.isRequired,
-  localPhaseItem: PropTypes.number.isRequired,
-  members: PropTypes.array.isRequired,
-  projects: PropTypes.object.isRequired,
-  queryKey: PropTypes.string,
-  team: PropTypes.object.isRequired
-};
+  componentDidMount() {
```

Comment:
@mattkrick I’ve gone back and forth, but I’m missing something trying to delay an update to state using `setTimeout` — any quick pointers?

Reply:
in crude untested form, it should look like this 
in the constructor:
`this. state = {hasWaited: false, showEmpty: false}`

in componentWillMount:
`setTimeout(() => {this.setState({hasWaited: true})}, 1000)`

in cWRP:
` if (this.state.hasWaited && noProjectsExistInObject(projects) { this.setState({showEmpty: true})`

in render:
`if (showEmpty) {renderEmptyStuff() } else { renderProjects()}`